.devcontainer/Dockerfile

```
@@ -0,0 +1,9 @@
+FROM node:18
```

what about an evergreen tag like:

```suggestion
FROM node:latest
```

---

We might also use `lts` might be safer